addons/analytic/models/analytic_distribution_model.py

```diff
@@ -6,17 +6,14 @@
 from odoo.exceptions import UserError
 
 
-class NonMatchingDistribution(Exception):
-    pass
-
-
 class AccountAnalyticDistributionModel(models.Model):
     _name = 'account.analytic.distribution.model'
     _inherit = 'analytic.mixin'
     _description = 'Analytic Distribution Model'
     _rec_name = 'create_date'
-    _order = 'id desc'
+    _order = 'sequence, id desc'
 
+    sequence = fields.Integer(default=10)
```

Comment:
after manual sequencing the lowest ID should be 10.
So saving a model from the widget will give it sequence 10. Since it has a higher ID, it will take priority

Reply:
Yes, but is it what we want :thinking: 
Just wondering if this was given a thought